### PR TITLE
Entity to person

### PR DESCRIPTION
**Pull request by tschmidtb51**
- resolves oasis-tcs/csaf#544
- change entity to person

### CODE REVIEW

**Review by santosomar — CHANGES_REQUESTED** [review]
As per my previous comment, in some cases acknowledgements can be done to an organization vs a person.  Examples at: https://msrc.microsoft.com/update-guide/acknowledgement/

I believe that it should either be left as an "entity" or put "person or organization".

**Comment by tschmidtb51** [comment]
@santosomar: There might be a misunderstanding here. When an organization is acknowledged that should be done through `organization`.

```
    "acknowledgments": [
      {
        "organization": "National Security Agency"
      },
      {
        "names": [
          "Oliver Lyak (@ly4k_)"
        ],
        "organization": "Trend Micro Zero Day Initiative"
      }
    ]
```

What did I miss?

File: csaf_2.0/prose/csaf-v2-editor-draft.md [hunk 1]
```diff
@@ -565,8 +565,8 @@ The properties are: `names`, `organization`, `summary`, and `urls`.
 
 #### 3.1.1.1 Acknowledgments Type - Names
 
-List of acknowledged names (`names`) has value type `array` with 1 or more items holds the names of entities being recognized.
-Every such item of value type `string` with 1 or more characters represents the name of entity being recognized and contains the name of a single person.
+List of acknowledged names (`names`) has value type `array` with 1 or more items holds the names of persons being recognized.
```
Comment (sthagen) [inline csaf_2.0/prose/csaf-v2-editor-draft.md:568]: Or, `contributors`? … the plural of person reads not so good to me, esp. when lacking a relating `the` - maybe my acquired language module just fails me here.

File: csaf_2.0/prose/csaf-v2-editor-draft.md [hunk 2]
```diff
@@ -565,8 +565,8 @@ The properties are: `names`, `organization`, `summary`, and `urls`.
...
+Every such item of value type `string` with 1 or more characters represents the name of person being recognized and contains the name of a single person.
```
Comment (sthagen) [inline csaf_2.0/prose/csaf-v2-editor-draft.md:569]: I would add a `the` and would also consider the term `contributor` - that would in my perception nicely include any natural or _juristical_ (as we say in German, lacking a translation here) person. Shortly, any entity that can be named.

File: csaf_2.0/json_schema/csaf_json_schema.json [hunk 3]
```diff
@@ -18,11 +18,11 @@
         "properties": {
           "names": {
             "title": "List of acknowledged names",
-            "description": "Contains the names of entities being recognized.",
+            "description": "Contains the names of persons being recognized.",
```
Comment (sthagen) [inline csaf_2.0/json_schema/csaf_json_schema.json:21]: See my nit comments in the prose, in short: Maybe use the term `contributor` as core concept?

**Review by sthagen — APPROVED** [review]

**Comment by santosomar** [comment]
Oh. You are completely correct! I thought that the "person" was referring to both fields. Thank you so much for the clarification.

**Review by santosomar — APPROVED** [review]

File: csaf_2.0/prose/csaf-v2-editor-draft.md (hunk 2 above)
Comment (tschmidtb51) [reply csaf_2.0/prose/csaf-v2-editor-draft.md]: I added the `the`. I did not change person to contributor (yet) as organizations should be listed in `organization`. If I'm not missing anything (right now), then there shouldn't be a need to cover legal persons (even though they are also persons...)

**Comment by santosomar** [comment]
The TC has approved this pull request during the OASIS CSAF Meeting on May 18th, 2022 -- with the suggestion to change the word "person" to "contributor(s)".

**Comment by tschmidtb51** [comment]
@santosomar: Please review and merge.

**Review by santosomar — APPROVED** [review]

**Pull request merged**